lesson-1-intro-to-k8s/README.md

```diff
@@ -50,11 +52,12 @@ Welcome to the first lesson of the Red Hat Czech's OpenShift course for beginner
 
 ## Intro
 
-* The difference between OpenShift and Kubernetes.
-* OpenShift can feel complex and intimidating but don't worry! With some effort it will deliver.
+* The difference between [OpenShift](https://github.com/openshift/origin) and [Kubernetes](https://github.com/kubernetes/kubernetes).
```

Comment:
https://github.com/openshift/origin is a test repo

Reply:
oki, I'll use the root of docs then :) https://docs.openshift.com/